File: power_of_two_double_minus.java
public class power_of_two_double_minus {
    
    public static void main(String[] args){
        double n = 1;

        for (int i = 0; i >= -1073; i--){
            n *= 0.5;

            System.out.println("2の" + i + "乗は" + n);
        }
    }
}

File: power_of_two_double_minus.cs
using System;
namespace power_of_two_double_minus
{
    class Program
    {
        static void Main(string[] args)
        {
            double n = 1;
            for (int i = 1; i >= -1072; i--)
            {
                n *= 0.5;
                Console.WriteLine(n);
            }
        }
    }
}